addons/event/models/event_registration.py

```diff
@@ -282,6 +282,7 @@ def action_send_badge_email(self):
             default_template_id=template and template.id,
             default_composition_mode='comment',
             default_email_layout_xmlid="mail.mail_notification_light",
+            name_with_seats_availability=False
```

Comment:
It's good to leave a trailing comma so the next time we add a key we don't need to modify this line (keeps blame history leaner)
```suggestion
            name_with_seats_availability=False,
```